### DISCUSSION TITLE
What options does setValue() have

### DISCUSSION THREAD

**Discussion by Pafnodeveloper** [webdriverio/webdriverio#7606] [Q&A]
I have an array of strings and I need to insert every string from it into a testArea with a new line. Is there any option in setValue method to do that cause I don't want to iterate every element from the array and add '\n'

**Comment by christian-bromann** [comment]
As mentioned in the [docs](https://webdriver.io/docs/api/element/setValue) you can pass in a value as array, e.g.:

```
await $('selector').setValue(['line1', '\n', 'line2', '\n', 'line3'])
```

**Reply by Pafnodeveloper** [reply]
And that is a manual work which I don't want to do, imagine if I have 10000 lines, so I need to write 10000 '\n' or iterate over the array  with loop and add '\n' at the end of every single line. It will be more practical to expand setValue method with more options
